lib/web_ui/lib/src/engine/safe_browser_api.dart

```diff
@@ -1033,18 +1034,18 @@ class OffScreenCanvas {
   Future<String> toDataUrl() {
     final Completer<String> completer = Completer<String>();
     if (offScreenCanvas != null) {
-      offScreenCanvas!.convertToBlob().then((html.Blob value) {
-        final html.FileReader fileReader = html.FileReader();
-        fileReader.onLoad.listen((html.ProgressEvent event) {
+      offScreenCanvas!.convertToBlob().then((DomBlob value) {
+        final DomFileReader fileReader = createDomFileReader();
+        fileReader.addEventListener('progress', allowInterop((DomEvent event) {
```

Comment:
Shouldn't we be listening to the `load` event here, not the `progress` event?

Reply:
good catch! yes, that was definitely a typo.